### PR TITLE
[CUA-2020] - Add DevopsDays Criciuma

### PR DESCRIPTION
**Pull request by guilhermealano**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by guilhermealano** [comment]
Pull request by Guilherme Alano

**Comment by bridgetkromhout** [comment]
Hi, @guilhermealano - your local organizer team needs to be listed in this initial PR, and we need their emails to set up the mail alias and Slack. (You can send their emails to info@devopsdays.org.)

**Comment by guilhermealano** [comment]
Hello, the PR was not approved? Should I submit a new one?

File: data/events/2020-cricima.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  - name: "John Doe"
+  - name: "Jane Smith"
+    twitter: "devopsdays"
+  - name: "Sally Fields"
+    employer: "Acme Anvil Co."
+    github: "devopsdays"
+    facebook: "https://www.facebook.com/sally.fields"
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "https://mattstratton.com"
+    image: "sally-fields.jpg"
```
Comment (bridgetkromhout) [inline data/events/2020-cricima.yml]: In this section, please put the names of your team.

File: data/events/2020-criciuma.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - name: "John Doe"
+  - name: "Jane Smith"
+    twitter: "devopsdays"
+  - name: "Sally Fields"
+    employer: "Acme Anvil Co."
+    github: "devopsdays"
+    facebook: "https://www.facebook.com/sally.fields"
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "https://mattstratton.com"
+    image: "sally-fields.jpg"
```
Comment (bridgetkromhout) [inline data/events/2020-criciuma.yml]: We need the names of your team members in this section.

**Comment by bridgetkromhout** [comment]
> Hello, the PR was not approved? Should I submit a new one?

Hi, @guilhermealano - 

You emailed us some email addresses. We need you to add those people's names in the section of data/events/2020-criciuma.yml that lists organizers (as explained above).

**Comment by guilhermealano** [comment]
Ok. Should I resubmit a new PR?

**Comment by bridgetkromhout** [comment]
> Ok. Should I resubmit a new PR?

I fixed problems in this PR. So you should edit this PR, not submit a new one. You should edit this file in this place: https://github.com/guilhermealano/devopsdays-web/blob/upstream/data/events/2020-criciuma.yml#L51-L61

Replace the examples with your real team members.

**Comment by guilhermealano** [comment]
#9421

**Pull request closed without merging**